addon/components/new-osf-navbar/template.hbs

```diff
@@ -37,7 +37,7 @@
                 <ul class="nav navbar-nav">
                     {{!SECONDARY NAVIGATION - displays corresponding links for selected service}}
                     {{!Pass in an observable value here so this will actually get rerendered}}
-                    {{#each (build-secondary-nav-links currentService currentUser.id) as |navLink|}}
+                    {{#each (build-secondary-nav-links currentService serviceSubmitLink) as |navLink|}}
```

Comment:
I was going to ask you to add `currentUser.id` back in, since it's the observable value mentioned in the comment, but I think #8/#9 actually worked around the need for that. Could you remove that comment?